### ISSUE TITLE
Improve clarity of CAPTCHAs

### ISSUE THREAD

**Issue by WilliamWDTK** [mysociety/alaveteli#6907]
A user, when contacting us through [the contact form on WhatDoTheyKnow](https://www.whatdotheyknow.com/help/contact), mentioned in a [postscript](https://www.lexico.com/definition/postscript) that they have eyesight problems, and that "[our] images for verification aren't clear enough for [them] to see well".  I responded by saying:

> Thank you for your feedback about our images for verification.  I assume you mean the CAPTCHAs that we have on our contact form, a bit like the one below:

(and I included the same image that I've included on this ticket).

The user then responded to say:

> Anyway, I obviously have eyesight problems.  Your traffic light below is not a problem.

I note that Google's reCAPTCHA (which is what we use) does normally offer an audio challenge, but this is perhaps not very obvious.  I tried again to activate the CAPTCHA, but it didn't appear, I don't know whether it is based on detected risk or not.  Here is an example from yesterday:

![Google's reCAPTCHA on WhatDoTheyKnow's contact form](https://user-images.githubusercontent.com/99501342/160191891-490d32af-d9cd-4552-be5e-54a50218377b.png)

**Comment by HelenWDTK** [comment]
This issue has been automatically closed due to a lack of discussion or resolution for over 12 months.
Should we decide to revisit this issue in the future, it can be reopened.
